src/components/widget.js

```diff
@@ -148,7 +148,10 @@ class Widget extends LitElement {
     return html`
       <div class="px-4 py-5">
         <section class="flex gap-4">
-          <obe-avatar avatar-url="${this.member.avatarUrl}" fallback="${this._getMemberInitials()}"></obe-avatar>
+          <obe-avatar
```

Comment:
What do you think about adding a `.prettierrc` file in a follow-up PR? I feel like we’re playing ping pong with line returns :D

Reply:
True enough 😵‍💫 Will add it to the post-release fixes checklist